assets/javascripts/kitten/components/atoms/typography/text/index.js

```diff
@@ -34,10 +35,15 @@ export const Text = ({
       'k-u-decoration-underline': decoration == 'underline',
       'k-u-decoration-none': decoration == 'none',
 
+      // Letter spacing.
+      'k-u-letter-spacing-10': letterSpacing == '10%',
```

Comment:
Et il se passe quoi si je veux du 20% ?
d'ailleurs, c'est un peu bizarre de mettre un letter-spacing en %, non?
J'ai plus en tête l'usage de `px`

Reply:
malheureusement pour tes attentes, les specs sont en pourcentage. Si tu désires 20%, tu peux ajouter la classe utilitaire via la partie SASS de Kitten, puis ajouter la prop `20%`. Sinon tu surcharges les styles avec Styled Components

![image](https://user-images.githubusercontent.com/1781070/152568664-b808cc89-3bde-4c09-88d5-545df36cf4ca.png)

Reply:
Du coup je suis perdu sur la notion de pourcentage ici. C'est un pourcentage de quoi?

Reply:
Jusqu'à preuve du contraire c'est en pourcentage d'`em`

Reply:
Pour l'ajout de la classe utilitaire, on ne pourrait pas plutôt passer directement la valeur de la props `letterSpacing` dans le style via une interpolation? Ca simplifierai un peu le tout, non?

Reply:
en copiant le code depuis Figma, il me donne `letter-spacing: 0.1em;` pour cet élément qui a un letter-spacing de `10%`

Reply:
Apparemment le % n'est pas un besoin produit ou design mais un "problème" de Figma:
https://forum.figma.com/t/letter-spacing-should-not-be-percentage-based/3062

Personnellement, je trouve très bizarre d'utiliser un pourcentage pour une valeur relative à une police.
(Ca relance aussi le débat sur les bizzareries de Figma par rapport à Abstract...)

Reply:
Le contexte de Kitten, la source de vérité, c'est la maquette. L'idée est de se détacher des formes imposées par le code pour se rapprocher le plus possible des specs. C'est la philosophie derrière les tokens par exemple, ou le fait de nommer les différents statuts des tags par exemple, sans rapport avec ce que le navigateur attend. En entrée on a des props en rapport direct avec les specs et les maquettes, en sortie on a du code valide et utilisable par le navigateur. Ça simplifie le travail d'intégration front : tu n'as pas à convertir la valeur en pourcentage vers une valeur en em, juste à remplir la valeur dans le composant `Text` (après, ça marche pas encore avec toutes les props—voir les noms donnés aux valeurs de graisse, mais ça c'est des trucs hérités qu'on n'a pas encore eu le temps d'améliorer)

Pour ce qui est des pourcentages, le `line-height` accepte des valeurs relatives (pourcentage, em) ou des valeurs absolues (px, cm, etc…), donc c'est pas choquant d'utiliser une valeur en pourcentage et la transcrire dans une autre valeur relative (em) pour le cas présent.